### PR TITLE
Spec file fixes

### PR DESCRIPTION
**Pull request by pevogam**
This fixes some missed variable changes and compatibility renaming details

### CODE REVIEW

File: avocado-plugins-vt.spec [hunk 1]
```diff
@@ -153,8 +153,8 @@ Xunit output, among others.
 %if %{with_python3}
 %files -n python3-%{name}
 %defattr(-,root,root,-)
-%dir /etc/avocado
-%dir /etc/avocado/conf.d
+%dir %{_sysconfdir}/avocado
+%dir %{_sysconfdir}/avocado/conf.d
```
Comment (ldoktor) [inline avocado-plugins-vt.spec]: Good point, would you please include the changelog entry for this? (use the same entry as in the second commit, and in the second commit add another item.

File: avocado-plugins-vt.spec [hunk 2]
```diff
@@ -167,6 +167,9 @@ Xunit output, among others.
 
 
 %changelog
+* Mon Sep 30 2019 Plamen Dimitrov <pdimitrov@pevogam.com> - 72.0-1
+- Rename package to "avocado_framework_plugin_vt"
```
Comment (ldoktor) [inline avocado-plugins-vt.spec]: Please increase the Version entry

File: avocado-plugins-vt.spec (hunk 1 above)
Comment (pevogam) [reply avocado-plugins-vt.spec]: Alright

File: avocado-plugins-vt.spec (hunk 2 above)
Comment (pevogam) [reply avocado-plugins-vt.spec]: Alright

**Comment by pevogam** [comment]
> Thank you, @pevogam, the only remaining things are the missing entry about the "/etc" and increased minor version.

Hi @ldoktor, I have pushed your required changes just now. One quick things that might be related to the spec file and we could probably add a fix on here:

In our plugin's spec we have Avocado VT as a dependency and encounter the following error:

```
Error: 
 Problem 1: conflicting requests
  - nothing provides python3.7dist(avocado-plugins-vt) needed by python3-avocado-plugins-i2n-72.0-1.fc30.noarch
 Problem 2: package intra2net-qa-deps-6.6.0-3.fc30.noarch requires python3-avocado-plugins-i2n = 72.0, but none of the providers can be installed
  - conflicting requests
  - nothing provides python3.7dist(avocado-plugins-vt) needed by python3-avocado-plugins-i2n-72.0-1.fc30.noarch
(try to add '--skip-broken' to skip uninstallable packages)
```

In our spec file we have "Requires: python3-aexpect, python3-avocado-plugins-vt" as an entry. This used to work before. Any idea if this is related to the recent transition to python 3.7 on the side of avocado?

**Comment by ldoktor** [comment]
Hello @pevogam, I don't really understand what doesn't work for you, anyway I tested this version and there is also the `setup.py` that needs to be adjusted (`make rpm` is failing for me).

While on that I did a brief `fgrep avocado-plugins-vt * -r` and it unveiled many outdated instances. Would you please also take care of these (documentation, virttest/version.py, ...).

**Comment by pevogam** [comment]
Hi, this commit worked just fine and was even approved for merging but now has conflicts with the newest master. @clebergnu as you were the pending reviewer for the commit I hope you don't mind accommodating for these conflicts.

**Comment by pevogam** [comment]
Thanks a lot @clebergnu for looking into this before the release! Much appreciated!

**Pull request merged**